docs/README.cs.md

```diff
@@ -0,0 +1,30 @@
+<h3 align="center"><img src="../assets/onefetch.svg" height="130px"></h3>
+
+<h5 align="center">Informační nástroj Git pro příkazový řádek vytvořený v Rust</h5>
+
+<p align="center">
+	<a href="https://crates.io/crates/onefetch"><img src="https://img.shields.io/crates/v/onefetch.svg" alt="cargo"></a>
+	<a href="https://github.com/o2sh/onefetch/actions"><img src="https://github.com/o2sh/onefetch/workflows/CI/badge.svg" alt="Build Status"></a>
+  <a href="https://onefetch.dev"><img src="../assets/language-badge.svg"></a>
+  <a href="https://github.com/o2sh/onefetch/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22"><img src="https://img.shields.io/github/issues/o2sh/onefetch/help%20wanted?color=green" alt="help wanted"></a>
+	<a href="../LICENSE.md"><img src="https://img.shields.io/badge/license-MIT-blue.svg"></a>
+	<img src="../assets/msrv-badge.svg">
+</p>
+
+<img src="../assets/screenshot-1.png" align="right" height="250px">
+
+Onefetch je informační nástroj Git pro příkazový řádek napsaný v `Rust`, který zobrazuje informace o projektu a statistiky kódu pro lokální úložiště Git přímo ve vašem terminálu. Nástroj je zcela offline – není vyžadován žádný přístup k síti.
+
+Ve výchozím nastavení se informace o úložišti zobrazují vedle loga dominantního jazyka, ale můžete si onefetch nakonfigurovat tak, aby místo toho používal obrázek (na podporovaných terminálech), textový vstup nebo vůbec nic.
+
+Automaticky detekuje licence open source z textů a poskytuje uživateli cenné informace, jako je distribuce kódu, čekající změny, počet závislostí (podle správce balíčků), hlavní přispěvatelé (podle počtu potvrzení), velikost na disku, datum vytvoření, počet řádků kódu atd.
+
+<img src="../assets/screenshot-2.png" align="right" height="250px">
+
+Onefetch lze nakonfigurovat tak, aby zobrazoval přesně to, co chcete, způsobem, jakým to chcete: můžete přizpůsobit formátování ASCII/Text, zakázat informační řádky, ignorovat soubory a adresáře, výstup ve více formátech (Json, Yaml), atd.
+
+V současné době podporuje onefetch více než [100 různých programovacích jazyků](https://onefetch.dev); pokud váš vybraný jazyk není podporován: Otevřete nový požadavek a podpora bude přidána.
+
+Příspěvky jsou vítány! Další informace najdete v [ZAPOJIT SE](CONTRIBUTING.md).
```

Comment:
We haven't been translating `CONTRIBUTING`, usually. See some of the other translations.